ckanext/switzerland/helpers/frontend_helpers.py

```diff
@@ -79,23 +79,37 @@ def localize_json_facet_title(facet_item):
 
 def get_frequency_name(identifier=None, get_map=False):
     frequencies = OrderedDict([
-        ('http://purl.org/cld/freq/irregular', _('Irregular')),  # noqa
-        ('http://purl.org/cld/freq/continuous', _('Continuous')),  # noqa
-        ('http://purl.org/cld/freq/daily', _('Daily')),  # noqa
-        ('http://purl.org/cld/freq/threeTimesAWeek', _('Three times a week')),  # noqa
-        ('http://purl.org/cld/freq/semiweekly', _('Semi weekly')),  # noqa
-        ('http://purl.org/cld/freq/weekly', _('Weekly')),  # noqa
-        ('http://purl.org/cld/freq/threeTimesAMonth', _('Three times a month')),  # noqa
-        ('http://purl.org/cld/freq/biweekly', _('Biweekly')),  # noqa
-        ('http://purl.org/cld/freq/semimonthly', _('Semimonthly')),  # noqa
-        ('http://purl.org/cld/freq/monthly', _('Monthly')),  # noqa
-        ('http://purl.org/cld/freq/bimonthly', _('Bimonthly')),  # noqa
-        ('http://purl.org/cld/freq/quarterly', _('Quarterly')),  # noqa
-        ('http://purl.org/cld/freq/threeTimesAYear', _('Three times a year')),  # noqa
-        ('http://purl.org/cld/freq/semiannual', _('Semi Annual')),  # noqa
-        ('http://purl.org/cld/freq/annual', _('Annual')),  # noqa
-        ('http://purl.org/cld/freq/biennial', _('Biennial')),  # noqa
-        ('http://purl.org/cld/freq/triennial', _('Triennial')),  # noqa
```

Comment:
Before the terms were all written beginning with an uppercase letter. AFAIK these terms are used for displaying. Was it requested to lower-case the terms?